corehq/sql_accessors/sql_templates/get_case_ids_in_domain_by_owners.sql

```diff
@@ -1,22 +0,0 @@
-DROP FUNCTION IF EXISTS get_case_ids_in_domain_by_owners(text, text[], boolean);
```

Comment:
should we not also drop this function in a migration that will run after this is merged?